testing/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/AndroidSantaTrackerSmokeTest.groovy

```diff
@@ -151,7 +151,13 @@ class AndroidSantaTrackerLintSmokeTest extends AndroidSantaTrackerSmokeTest {
 }
 
 class SantaTrackerConfigurationCacheWorkaround {
+
     static void beforeBuild(File checkoutDir, File gradleHome) {
+        String androidSdkRootPath = System.getenv("ANDROID_SDK_ROOT")
```

Comment:
Could you use/update `AndroidHome.assertIsSet()`?

Reply:
Thank you for the tip!